content/en/docs/installation.md

```diff
@@ -112,25 +112,74 @@ Going forward, we'll continue to support 10 most recent versions of Minikube wit
 
 ### GKE
 
-Google Kubernetes Engine (GKE) uses Container-Optimized OS (COS) as the default operating system for its worker node pools. COS is a security-enhanced operating system that limits access to certain parts of the underlying OS. Because of this security constraint, Falco cannot insert its kernel module to process events for system calls. However, COS provides the ability to leverage eBPF (extended Berkeley Packet Filter) to supply the stream of system calls to the Falco engine. eBPF is currently supported only on GKE and COS.
+Google Kubernetes Engine (GKE) uses Container-Optimized OS (COS) as the default operating system for its worker node pools. COS is a security-enhanced operating system that limits access to certain parts of the underlying OS. Because of this security constraint, Falco cannot insert its kernel module to process events for system calls. However, COS provides the ability to leverage eBPF (extended Berkeley Packet Filter) to supply the stream of system calls to the Falco engine.
 
-#### Enabling eBPF Support
 
-Falco can use eBPF with minimal configuration changes. To do so, set the `SYSDIG_BPF_PROBE` environment variable to an empty value: `SYSDIG_BPF_PROBE=""`. Setting this environment variable will trigger the `falco-probe-loader` script to download the kernel headers for the appropriate version of COS, and then compile the appropriate eBPF probe. Alternatively, you can set `SYSDIG_BPF_PROBE` to the path of an existing eBPF probe.
+## Enabling eBPF Support
 
+Falco can use eBPF with minimal configuration changes. To do so, set the `FALCO_BPF_PROBE` environment variable to an empty value: `FALCO_BPF_PROBE=""`.
+
+eBPF is currently supported only on GKE and COS, however here we provide installation details for a wider set of platforms
+
+**IMPORTANT**: If you want to specify an alternative path for the probe file, you can also set `FALCO_BPF_PROBE` to the path of an existing eBPF probe.
```

Comment:
If you want to specify an alternative path for the probe file, set `FALCO_BPF_PROBE` to the path of an existing eBPF probe.